src/OrchardCore/OrchardCore/OrchardCore.csproj

```diff
@@ -15,6 +15,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\OrchardCore.Abstractions\OrchardCore.Abstractions.csproj" />
+    <ProjectReference Include="..\OrchardCore.Localization.Abstractions\OrchardCore.Localization.Abstractions.csproj" />
```

Comment:
Why? I expect `OrchardCore.Abstractions` may reference `OrchardCore.Localization.Abstractions`

Reply:
OrchardCore.Abstractions doesn't need it, so why do it?